src/meltano/core/project.py

```diff
@@ -231,9 +231,23 @@ def meltano(self) -> MeltanoFileHint:
             the current meltano config
         """
         from .meltano_file import MeltanoFile
+        from .settings_service import FEATURE_FLAG_PREFIX, FeatureFlags
+        from .yaml import configure_yaml
 
-        with self._meltano_rw_lock.read_lock():
-            return MeltanoFile.parse(self.project_files.load())
+        with open(self.meltanofile) as melt_ff:
+            meltano_ff: dict = configure_yaml().load(melt_ff)
+
+        uvicron_enabled = (
```

Comment:
```suggestion
        uvicorn_enabled = (
```

Reply:
@cjohnhanson thanks for catching that.  I made and committed the suggested changes.